OscarsGame.Web/CommonPages/TermsOfService.aspx

```diff
@@ -0,0 +1,51 @@
+
+    <ul>
+        <li>modify or copy the materials;</li>
+        <li>use the materials for any commercial purpose or for any public display;</li>
+        <li>attempt to reverse engineer any software contained on Oscars Game's Website;</li>
+        <li>remove any copyright or other proprietary notations from the materials; or</li>
+        <li>transferring the materials to another person or "mirror" the materials on any other server.</li>
+    </ul>
+
+    <p>This will let Oscars Game to terminate upon violations of any of these restrictions. Upon termination, your viewing right will also be terminated and you should destroy any downloaded materials in your possession whether it is printed or electronic format. These Terms of Service has been created with the help of the <a href="https://www.termsofservicegenerator.net">Terms Of Service Generator</a> and the <a href="https://www.generateprivacypolicy.com">Privacy Policy Generator</a>.</p>
+
+    <h2>3. Disclaimer</h2>
+
+    <p>All the materials on Oscars Game’s Website are provided "as is". Oscars Game makes no warranties, may it be expressed or implied, therefore negates all other warranties. Furthermore, Oscars Game does not make any representations concerning the accuracy or reliability of the use of the materials on its Website or otherwise relating to such materials or any sites linked to this Website.</p>
+
+    <h2>4. Limitations</h2>
+
+    <p>Oscars Game or its suppliers will not be hold accountable for any damages that will arise with the use or inability to use the materials on Oscars Game’s Website, even if Oscars Game or an authorize representative of this Website has been notified, orally or written, of the possibility of such damage. Some jurisdiction does not allow limitations on implied warranties or limitations of liability for incidental damages, these limitations may not apply to you.</p>
+
+    <h2>5. Revisions and Errata</h2>
+
+    <p>The materials appearing on Oscars Game’s Website may include technical, typographical, or photographic errors. Oscars Game will not promise that any of the materials in this Website are accurate, complete, or current. Oscars Game may change the materials contained on its Website at any time without notice. Oscars Game does not make any commitment to update the materials.</p>
+
+    <h2>6. Links</h2>
+
+    <p>Oscars Game has not reviewed all of the sites linked to its Website and is not responsible for the contents of any such linked site. The presence of any link does not imply endorsement by Oscars Game of the site. The use of any linked website is at the user’s own risk.</p>
+
+    <h2>7. Site Terms of Use Modifications</h2>
+
+    <p>Oscars Game may revise these Terms of Use for its Website at any time without prior notice. By using this Website, you are agreeing to be bound by the current version of these Terms and Conditions of Use.</p>
+
+    <h2>8. Your Privacy</h2>
+
+    <p>Please read our Privacy Policy.</p>
```

Comment:
Should this be a link to the privacy policy page?

Reply:
Yes, actually it's a good idea :)

Reply:
Done :)